man/man1/freight-add.1

```diff
@@ -26,6 +26,10 @@ Use an alternate configuration file\.
 Verbose mode\.
 .
 .TP
+\fB\-e\fR, \fB\-\-add\-error\fR
```

Comment:
No. I did not see in the makefile that the files are generated. So I found a bit obscure modifying directly the man files.
The .ronn file is modified and pushed again.